pyAMISR/__init__.py

```diff
@@ -15,6 +15,10 @@
 
 """
 try:
-	from analyze import *
-except Exception, e:
-    print 'Problem importing from analyze.py: ' + str(e)
\ No newline at end of file
+    from pathlib import Path
```

Comment:
"pathlib" is a python3 built-in (backported to python2) that allows for high-level, object-oriented manipulation of paths. Instead of 80+ columns of:


    try:
        os.mkdir(os.path.expanduser(mypath))
    except OSError:
        pass
    
    mypath + os.pathsep + os.path.split(os.path.expanduser(mynewfile))[0] + os.path.split(os.path.expanduser(myfile))[1]

you just do
    
    mypath.mkdir(exists_ok=True)
    fn = mypath / (mynewfile.name.with_suffix(myfile.suffix))

and numerous more easy methods of iterating over files & directories with recursive globbing built-in, etc.